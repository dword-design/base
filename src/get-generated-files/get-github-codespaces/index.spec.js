import { Base } from '@/src/index.js';

export default {
  works() {
    const base = new Base({ nodeVersion: 18 });
    expect(base.getGithubCodespacesConfig()).toMatchSnapshot(this);
  },
};
