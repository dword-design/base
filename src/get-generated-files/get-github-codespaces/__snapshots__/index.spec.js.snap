// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index works 1`] = `
Object {
  "features": Object {
    "ghcr.io/devcontainers/features/docker-in-docker:2": Object {},
  },
  "image": "mcr.microsoft.com/devcontainers/javascript-node:1-18",
  "updateContentCommand": "pnpm install --frozen-lockfile",
}
`;
